Feat/refactor html style

RitaCaterina opened the pull request:
after prettier corrected i remove style in line and used semantics tags

Commit:
chore:formattazione prettier registri.html

Commit:
refactor: remove style-inline

Commit:
refactor:aside with two buttons

Commit:
refactor:view register and menu edit sections

Commit:
refactor: preliminary registri.css

Aierbote reviewed (commented):
It's not ripe enough yet. Good work so far, I'll take it from here!

Aierbote committed:
refactor: define a custom default and color palette

Aierbote committed:
feat: saved the colors from the logo in vars

Aierbote reviewed (changes requested):
There is still alot to do

Rizaaal reviewed (dismissed):
The CSS is incomplete, but commit is merged anyways as we need the html to start working on frontend JS logic

Commit:
feat(css): inserted new CSS into existing CSS.

page style is incomplete, this commit serves the purpose of cleaning the CSS so that
we can push the HTML and start working on it.
